spark/src/test/scala/geotrellis/spark/io/cassandra/CassandraRasterCatalogSpec.scala

```diff
@@ -122,7 +144,8 @@ class CassandraRasterCatalogSpec extends FunSpec
           val actual = CoordinateSpaceTime.collect.toMap.apply(key)
           tilesEqual(tile, actual)
         }
-        
+        // FIXME: Expected exception geotrellis.spark.io.package$LayerNotFoundError to be thrown,
+        // but java.lang.RuntimeException was thrown. (CassandraRasterCatalogSpec.scala:148)
```

Comment:
These fixme comments were where the tests failed because the expected exception wasn't originally thrown in the implementing catalog class.

I'll remove all comments.

Sent from mobile

Rob Emanuele <notifications@github.com> wrote:

>In spark/src/test/scala/geotrellis/spark/io/cassandra/CassandraRasterCatalogSpec.scala:
>
>> @@ -122,7 +144,8 @@ class CassandraRasterCatalogSpec extends FunSpec > val actual = CoordinateSpaceTime.collect.toMap.apply(key) > tilesEqual(tile, actual) > } > - > + // FIXME: Expected exception geotrellis.spark.io.package$LayerNotFoundError to be thrown, > + // but java.lang.RuntimeException was thrown. (CassandraRasterCatalogSpec.scala:148) 
>
>?
>
>—
>Reply to this email directly or view it on GitHub.￼
>